packages/web-container/index.html.ejs

```diff
@@ -33,7 +34,8 @@
   <script>
     requirejs.config({
       baseUrl: <%- JSON.stringify(data.roots.js) %>,
-      paths: <%- JSON.stringify(data.bundle.js) %>
+      paths: <%- JSON.stringify(data.bundle.js) %>,
+      waitSeconds: 20
```

Comment:
is this still needed after your vendor.js minifications?

Reply:
In normal scenarios, doesn't look like. But network connections in the neighbouring france are still pretty stone age... Do you prefer I remove it?

Reply:
i would prefer to have this as build option